### ISSUE TITLE
Update hw_USBTin to use can232.py

### ISSUE THREAD

**Issue by DePierre** [CANToolz/CANToolz#28]
The `hw_USBTin` module is speaking LAWICEL to the hardware device. Since `hw_CAN232.py`, the LAWICEL protocol support has been implemented in `can232.py.

IMO the USBTin module should be updated to use `can232.py` instead of manually reimplementing the protocol support.

**Comment by igittigitt** [comment]
I agree, USBTins UCCB-Protocol is a subset of Lawicels SLCAN-Protocol. USBTin should better adopt it fully.
